python/fastquant/strategies/base.py

```diff
@@ -289,6 +294,33 @@ def next(self):
                         self.sell(exectype=bt.Order.StopTrail, trailpercent=self.stop_trail, size=final_size)
 
         elif self.sell_signal():
+            if self.allow_short == True:
+
+                # Sell short based on the closing price of the previous day
+                if self.execution_type == "close":
+
+                    if self.transaction_logging:
+                        self.log("SELL CREATE, %.2f" % self.dataclose[0])
+
+                    sell_prop_size = int(self.short_max * 
+                                         self.cash / 
+                                         self.dataclose[0] /
+                                         (1 + self.commission + 0.001))
```

Comment:
```suggestion
```

Reply:
No need to do this here. This is just for the buy. Commission is merely deducted from proceeds.